backend/auth.js

```diff
@@ -28,8 +27,8 @@ const init = oauthApi => {
           Authorization: `Basic ${apiClientCredentials(config.apis.oauth2.clientId, config.apis.oauth2.clientSecret)}`,
         },
       },
-      (accessToken, refreshToken, profile, done) =>
-        done(null, { access_token: accessToken, refresh_token: refreshToken })
+      (accessToken, refreshToken, params, profile, done) =>
```

Comment:
pass authSource instead of params.auth_source?

Reply:
not sure what you mean.  The `auth_source` parameter is taken out of `params` and passed in as `authSource` into the `done` method